Community.md

```diff
@@ -0,0 +1,47 @@
+# Community Continuity Guidelines v2.0.0
+
+As the creators, and maintainers of this project. We want to ensure that the project lives and continues to grow. Not blocked by any singular person's computer time.
+
+One of the simplest ways of doing this, is by encouraging a larger set of shallow contributors.
+
+#### Ownership
+
+If you get a merged PR, regardless of content (typos, code, doc fixes) then you're eligible for push access to this organization. We check for this on PR merges and extend an invite via GitHub.
+
+Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project. If you are working with a codebase, then you probably have the skills to improve it, and offer valuable feedback.
+
+Everyone comes in with their own perspective on what a project could/should look like, and encouraging discussion can help expose good ideas sooner.
+
+#### Why do we give out push access?
+
+It can be overwhelming to suddenly be offered the chance to wipe the source code for a project. Don't worry, we don't let you push to master.
+
+All code is peer-reviewed, and we have the convention that someone other than the submitter should merge the pull request.
+
+As an organization contributor you can merge other people's pull requests, or other contributors can merge yours. You won't be assigned a pull request, but you're welcome to just jump in and take a code review on things that interest you.
+
+If it feels right, merge.
+
+This project is not continuously deployed, there is space for debate after review too. Offering the chance to revert, or make an amending pull request.
+
+It's convention to provide a call to action via `/cc @[org]/contributors` to invite discussion.
+
+#### How can we help you get comfortable contributing?
+
+It's normal for a first PR to be a potential fix for a problem, moving on from there to helping the project's direction can be difficult. We try to help contributors cross that barrier by offering good first step issues. These issues can be done without feeling like you're stepping on toes. Ideally, these are non-critical issues that are well defined. They will be purposely avoided by mature contributors to the project.
+
+We aim keep all project discussion inside GitHub issues. This is to make sure it is public. If you have questions about how to use the library, or how the project is running GitHub issues are the goto tool.
```

Comment:
I think a comma is missing:

> …how to use the library, or how the project is running, GitHub issues are the goto tool.

And “how the project is running” sounds like a status update, do you mean “how the project is ran” ?